model/src/test/kotlin/JsonSchemaTest.kt

```diff
@@ -99,6 +99,19 @@ class JsonSchemaTest : StringSpec() {
 
             errors should beEmpty()
         }
+
+        "reference.yml validates successfully" {
+            val ortConfigurationSchema = JsonSchemaFactory
+                .builder(JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V6))
```

Comment:
Why is this `V6` instead of `V7` like the others?

Reply:
Which boils down why we use `"$schema": "http://json-schema.org/draft-06/schema#"` in the previous commit.

Reply:
The other schemas use features which are only available in `V7` (if-then-else: https://json-schema.org/understanding-json-schema/reference/conditionals.html#if-then-else).
As the other schemas are split and reuse common components, all of them have to be `V7`.

In this case, no features from `V7` are (yet) used, and therefore I did use `V6`, as per the [best practices](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices) the "lowest possible version should be used to ensure interoperability with as many supported editors, IDEs and parsers as possible."

But thanks to you for pointing this out, as we actually support `V4` which I now switched to.

Reply:
> as per the [best practices](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices) the "lowest possible version should be used

Makes sense. Is it worth to mention this in the commit message?